odoo/addons/base/models/ir_ui_view.py

```diff
@@ -194,7 +194,9 @@ def get_view_arch_from_file(filename, xmlid):
 xpath_utils = etree.FunctionNamespace(None)
 xpath_utils['hasclass'] = _hasclass
 
+TRANSLATED_ATTRS = [attr for attr in TRANSLATED_ATTRS if attr != 'src']
 TRANSLATED_ATTRS_RE = re.compile(r"@(%s)\b" % "|".join(TRANSLATED_ATTRS))
+TRANSLATED_IMG_SRC_RE = re.compile(r"(?<=img\[contains\()@src\b")  # Image with tranlatable src attribute
```

Comment:
What if the XPath expression uses another operator than `contains()`?
Perhaps something like this would match (reasonably) more cases: `img\[[^\]]*@src\b`?
Still not perfect, but it's impossible to cover 100% with a regex.

Also the inline comment is weird, it doesn't explain the regex?